requirements.txt

```diff
@@ -4,3 +4,5 @@ numpy >= 1.15.4
 mypy_extensions >= 0.4.1
 regex >= 2018.01.10
 sentencepiece >= 0.1.8
+
```

Comment:
We can add the requirement of Petuum-NNI when it is done. We can approve this PR but wait for the NNI released to merge

Reply:
Sure, will work on it after Petuum-NNI is released.

Reply:
The latest NNI version has been updated with adl training service, so it can be directly installed for our nni-adl example. I have update the requirement.txt to include this dependency.